app/models/user.rb

```diff
@@ -56,7 +56,7 @@ def otp_time_lockout?
 
   def lock_access!(opts = {})
     super
-    send_devise_notification(:unlock_instructions, nil, subject: 'Upaya Account Locked')
+    send_devise_notification(:unlock_instructions, nil, subject: 'Account Locked')
   end
```

Comment:
Hmm .. you're right, that was a different change.  I was thinking of removing the app name here, but that is for the UX team to decide.   I'll revert this for now.